embd.go

```diff
@@ -72,20 +78,50 @@ func run() error {
 		Args:  os.Args[1:],
 		Pkg:   *pkg,
 		Files: map[string]File{},
+		Dirs:  map[string]map[string]File{},
 	}
 	for _, path := range flag.Args() {
-		// TODO(akavel): support directories
-		f, err := NewFile(path)
+		path := filepath.ToSlash(path)
+		info, err := os.Stat(path)
 		if err != nil {
 			return err
 		}
-		if old, exists := contents.Files[f.VarName]; exists {
-			return fmt.Errorf(
-				"generated variable name conflict: '%s' resolves to"+
-					"the same variable name %s as '%s'",
-				f.Path, f.VarName, old.Path)
+
+		if info.IsDir() {
+			dir, err := ioutil.ReadDir(path)
+			if err != nil {
+				return err
+			}
+
+			k := "Dir" + normalize(path)
+			if _, exists := contents.Dirs[k]; exists {
+				return fmt.Errorf("directory %s was resolved to variable %s. But last's already found", path, info.Name())
+			}
+
+			files := map[string]File{}
+			for _, info := range dir {
+				f, err := NewFile(path + "/" + info.Name())
```

Comment:
This won't work for sub-directories, right? If so, then please:

1. Add a test here:

        if info.IsDir() {
            continue
        }
2. Add a line:

        // TODO(akavel): add subdirectories recursively
3. Add a note in flag.Usage (between PrintDefaults and Fprintln(os.Stderr, version)), e.g. "Note: directories are added non-recursively (only immediate children)."
4. Update the USAGE in the README file with the note.